app/jobs/appointment_reminder_job.rb

```diff
@@ -5,7 +5,7 @@ def perform(appointment)
     ReminderActivity.create!(
       appointment: appointment,
       owner: appointment.guider,
-      message: 'sent a reminder'
+      message: 'sent an appointment reminder email to the customer'
```

Comment:
Maybe just worth keeping this as it was (maybe we should have just provided an empty string for `message` from the start) and putting this message directly in the template instead? My view on these is it's more important to use `message` when computing the message at runtime is potentially expensive - such as when we compute the audited fields.